bundle/Helper/MailHelper.php

```diff
@@ -68,6 +68,47 @@ public function sendMail(array|string $receivers, string $subject, string $templ
         $this->mailer->send($email);
     }
 
+    /**
+     * Sends a group mail to users in bcc.
+     *
+     * Sender and recipient are set to sender
+     *
+     * Sender can be:
+     * a string: info@netgen.io
+     * an array: array( 'info@netgen.io' => 'Netgen Site' )
+     *
+     * Bcc can be:
+     *  a string: info@netgen.io
+     *  or:
+     *  array( 'info@netgen.io' => 'Netgen Site' ) or
+     *  array( 'info@netgen.io', 'example@netgen.io' ) or
+     *  array( 'info@netgen.io' => 'Netgen Site', 'example@netgen.io' => 'Example' )
+     *
+     * @param string|string[] $bcc
+     * @param array<string, mixed> $parameters
+     * @param string|string[]|null $sender
+     */
+    public function sendGroupMail(array|string $bcc, string $subject, string $template, array $parameters = [], array|string|null $sender = null): void
```

Comment:
What I would do here is not add a `sendGroupMail` that only sends to bcc, but add two new parameters at the end of `sendMail` method `ccReceivers` and `bccReceivers` which can then be used in conjuction with regular receivers.

It's a pretty common usecase to have a regular receiver and then a list of ccs and bccs.